usl_models/usl_models/flood_ml/model.py

```diff
@@ -376,75 +336,56 @@ def forward(
 
         return output
 
-    def call(self, inputs: dict[str, tf.Tensor]) -> tf.Tensor:
+    def call_n(self, full_input: Input, n: int = 1) -> tf.Tensor:
         """Runs the entire autoregressive model.
 
         Args:
             inputs: A dictionary of input tensors.
```

Comment:
Need to update the name here to `full_input`. Can you also explain how this differs from the `input` in the `call` method?

Reply:
Done.